modules/telestion-services/src/main/java/de/wuespace/telestion/services/loader/verticle/VerticleLoaderConfiguration.java

```diff
@@ -0,0 +1,7 @@
+package de.wuespace.telestion.services.loader.verticle;
+
+import de.wuespace.telestion.application.loader.LoaderConfiguration;
+
+public interface VerticleLoaderConfiguration extends LoaderConfiguration {
```

Comment:
Please let nobody make the song "We don't need no Java-Docs... 🎸" about our repository :D